pandas/tests/series/test_reductions.py

```diff
@@ -114,3 +114,23 @@ def test_validate_stat_keepdims():
     )
     with pytest.raises(ValueError, match=msg):
         np.sum(ser, keepdims=True)
+
+
+def test_idxmax_idxmin_with_nullable_integer_dtype():
+    ser = Series([1, 2, 3], dtype="Int64")
+    assert ser.idxmax() == 2
+    assert ser.idxmin() == 0
```

Comment:
Can you also add a test case with a missing value in the data? And we should probably also parameterize with skipna=True/False

Reply:
Thanks. Done.